moveit_ros/planning/planning_scene_monitor/include/moveit/planning_scene_monitor/current_state_monitor.h

```diff
@@ -99,6 +99,12 @@ class CurrentStateMonitor
    */
   bool haveCompleteState() const;
 
+  /** @brief Query whether we have joint state information for all DOFs in the kinematic model
+   *  @param min_update_time All joint information must be from this time or more current
+   *  @return False if we have no joint state information for one of the joints or if our state
+   *  information is more than \e age old*/
+  bool haveCompleteState(const ros::Time& min_update_time) const;
```

Comment:
```suggestion
  bool haveCompleteState(const ros::Time& oldest_allowed_update_time) const;
```
Same for the other occurrences. More verbosity, more good.